test/quantization/core/test_quantized_op.py

```diff
@@ -3740,6 +3740,117 @@ def test_qlinear_unpack(self, W, use_channelwise):
             np.testing.assert_equal(
                 W_q.q_zero_point(), W_q_origin.q_zero_point())
 
+    """Tests the correctness of the quantized linear_leaky_relu op."""
+    @given(batch_size=st.integers(1, 4),
+           input_channels=st.integers(16, 32),
+           output_channels=st.integers(4, 8),
+           use_bias=st.booleans(),
+           use_multi_dim_input=st.booleans(),
+           use_channelwise=st.booleans())
+    @override_qengines
+    def test_qlinear_leaky_relu(self, batch_size, input_channels, output_channels, use_bias,
+                                use_multi_dim_input, use_channelwise):
+        decimal_val = 4
+        dtype = torch.quint8
+        negative_slope = 0.01
+
+        nptype = np_dtype[dtype]
+        qlinear_prepack = torch.ops.quantized.linear_prepack
+        qlinear = torch.ops.quantized.linear_leaky_relu
+        if use_multi_dim_input:
+            batch_size *= 3  # Test the multi-dim input tensor
+        X_scale = 1.5
+        X_zp = 5
+        X_value_min = -128 if dtype == torch.qint8 else 0
+        X_value_max = 127 if dtype == torch.qint8 else 255
+        X_q0 = np.round(
+            np.random.rand(batch_size, input_channels) *
+            (X_value_max - X_value_min)
+            + X_value_min
+        ).astype(nptype)
```

Comment:
can we use `_quantize` function? https://github.com/pytorch/pytorch/blob/master/test/quantization/core/test_quantized_op.py#L92